phpunit/class-block-templates-test.php

```diff
@@ -74,6 +74,30 @@ function test_gutenberg_build_template_result_from_post() {
 		$this->assertEquals( 'wp_template', $template->type );
 	}
 
+	function test_inject_theme_attribute_in_content() {
+		$theme                           = get_stylesheet();
+		$content_without_theme_attribute = '<!-- wp:template-part {"slug":"header","align":"full", "tagName":"header","className":"site-header"} /-->';
+		$template_content                = _inject_theme_attribute_in_content(
+			$content_without_theme_attribute,
+			$theme
+		);
+		$this->assertStringContainsString( $theme, $template_content );
+
+		$content_with_existing_theme_attribute = '<!-- wp:template-part {"slug":"header","theme":"fake-theme","align":"full", "tagName":"header","className":"site-header"} /-->';
+		$template_content                      = _inject_theme_attribute_in_content(
+			$content_with_existing_theme_attribute,
+			$theme
+		);
+		$this->assertStringNotContainsString( $theme, $template_content );
+
+		$content_with_no_template_part = '<!-- wp:post-content /-->';
+		$template_content              = _inject_theme_attribute_in_content(
+			$content_with_existing_theme_attribute,
+			$theme
+		);
+		$this->assertStringNotContainsString( $theme, $template_content );
```

Comment:
I think we should assert `$template_content === $content_with_no_template_part`. We want to make sure the `_inject_theme_attribute_in_content` isn't making any changes to the markup.

Reply:
Makes sense 👍